composer.json

```diff
@@ -23,7 +23,7 @@
     },
     "require": {
         "php": "^7.1",
-        "symfony/console": "^3.2"
+        "symfony/console": "^3.2|^4.2"
```

Comment:
You just added version **^4** because you are using it?

Reply:
yep, working fine